### ISSUE TITLE
PS4 DS4 "Emulation" Mode and Xinput Mode

### ISSUE THREAD

**Issue by TurdFerguson64** [raphnet/gcn64tools#1]
Would it be possible to have/add a setting/feature in the "Raphnet adapter management tool" and firmware update to allow the "PS1/PS2 controller to USB adapter" to allow the attached controller(s) to act/be read as a PS4 DS4 controllers? This would allow the controller(s) to be usable with the "PS4 Remote Play" function, as well as to allow PS4 button prompts/icons to show up in steam!!!

Also, Would it be possible to have a setting, and updated firmware, for all, or most, of the controller adapters to be able to have/be in Xinput mode?

**Comment by raphnet** [comment]
Yeah, unfortunately I don't think we will ever implement an Xinput mode or a PS4 controller mode. It would be a lot to maintain.
